docs/development/testing.md

```diff
@@ -154,6 +154,23 @@ an `HTTP 400 error`.
 In that case, configure an alternate load method by referencing the name of the test BPMN diagram in the load configuration
 settings.
 
+### Performance tests
+For now these tests are defined under `/test/e2e/performance/` as the performance is being measured for complex(e2e) tasks:
+- display diagram
+- zoom in / zoom out loaded diagram
+
+#### Why?
+To have visibility if new changes have impact on current performance
+
+#### Results
+The results are under `/test/e2e/performance/data/`
+Preview is available in the browser, simply open file `/bpmn-visu-js/test/e2e/performance/index.html` in your favourite browser
+
+#### Run the performance tests
+For the moment we have decided to run them manually, in the future they may be run automatically - this however, depends on received results
+To run these tests, follow next steps:
+1. cd `bpmn-visu-js\test\e2e`
```

Comment:
I would only keep the path related to the root folder as we don't have the same folder name that store the repo locally. 
```suggestion
1. cd `./test/e2e`
```